<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_null_yunTaiDialog</class>
 <widget class="QDialog" name="dialog_null_yunTaiDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_yunTaiDialog_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>450</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>云台控制</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_yunTaiDialog_shutdown">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>0</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yunTaiDialog_center">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>29</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yunTaiDialog_down">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yunTaiDialog_right">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>85</y>
     <width>22</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yunTaiDialog_left">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>85</y>
     <width>22</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yunTaiDialog_up">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yunTaiDialog_apertureSub">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yunTaiDialog_apertureAdd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yunTaiDialog_aperture">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> 光圈</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yunTaiDialog_zoomedAdd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yunTaiDialog_zoomed">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> 变倍</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yunTaiDialog_zoomedSub">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yunTaiDialog_zoomingAdd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yunTaiDialog_zooming">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> 变焦</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yunTaiDialog_zoomingSub">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yunTaiDialog_channel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>通道</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yunTaiDialog_speed">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yunTaiDialog_Preset">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>预置点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yunTaiDialog_cruise">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>巡航线</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yunTaiDialog_more">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>更多</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_yunTaiDialog_channel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_yunTaiDialog_speed">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_yunTaiDialog_speedDisplay">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
